src/pyodide/internal/workers.py

```diff
@@ -53,16 +55,14 @@ class FetchKwargs(TypedDict, total=False):
 #       duplicates are lost, we should fix that so it returns a http.client.HTTPMessage
 class FetchResponse(pyodide.http.FetchResponse):
     # TODO: Consider upstreaming the `body` attribute
+    # TODO: Behind a compat flag make this return a native stream (StreamReader?), or perhaps
+    #       behind a different name, maybe `stream`?
     @property
-    def body(self) -> Body:
+    def body(self) -> "js.ReadableStream":
         """
-        Returns the body from the JavaScript Response instance.
+        Returns the body as a JavaScript ReadableStream from the JavaScript Response instance.
         """
-        b = self.js_response.body
-        if b.constructor.name == "FormData":
-            return FormData(b)
```

Comment:
Why did we have the special case for `FormData` before? And why are we removing it now?

Reply:
I was wondering the same thing and I couldn't recall why this was added. I don't think `body` can return anything other than ReadableStream so this code does nothing.